dvc/remote/http.py

```diff
@@ -23,7 +23,7 @@ def ask_password(host, user):
     )
 
 
-class HTTPRemoteTree(BaseRemoteTree):
+class HTTPRemoteTree(BaseRemoteTree):  # pylint:disable=abstract-method
```

Comment:
Pylint throws `abstract-method` as, `walk_files` and `get_file_hash` are not implemented for all. Not an issue.